recipes/jbig2dec/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jbig2dec" %}
+{% set version = "0.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs950/jbig2dec-{{ version }}.tar.gz
+  sha256: f72bad7102638b31fa96be7492fb3d447a83d71c644cffd01f2a7ec52bd5fb72
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('jbig2dec', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - zlib
+    - libpng
+
+test:
+  commands:
+    - jbig2dec --help
+
+about:
+  home: https://jbig2dec.com/
+  license: AGPL-3.0-only
+  license_family: GPL
```

Comment:
Is this the right family? AGPL is pretty different from the rest of GPL. I am surprised anaconda included it there.

Reply:
Oh there is an AGPL.  I wish this list was on the conda-build website, it is impossible to find. https://github.com/conda/conda-build/blob/master/conda_build/license_family.py#L8

Reply:
cc @jjhelmus @forrestwaters

Reply:
Also the repo data patches will autogenerate this field in principle